doc/guide/cert-authentication.xml

```diff
@@ -202,4 +202,83 @@ CPUQuota=30%
 
   </section>
 
+  <section id="certauth-forwarding">
+    <title>Authentication to other services like sudo and ssh</title>
+
+    <para>Once you logged into Cockpit with a certificate, you likely need to switch to administative mode
+      (root privileges through sudo), or connect to remote machines through SSH. If your user account has a password,
+      that can be used for authenticating to sudo or ssh as usual.</para>
+
+    <para><emphasis>Supported with FreeIPA only:</emphasis> If the user account does not have a
```

Comment:
You can also do this even if the account does have a password, no?  Maybe just write it like "As an alternative to password authentication, you can also..."

Reply:
Right. Thanks for the suggestion, applied.